docs/setup-project.adoc

```diff
@@ -0,0 +1,47 @@
+
+Hercules CI is configured "by convention" using `.nix` files in your repository.
+Follow these steps to ensure a correct setup.
+
+// TODO: If you haven't set up a project with Nix before, we recommend the <<tutorial>>
+
+1. Clone your repository
+
+2. Choose which Nix file to use for CI
++
+Reuse `/default.nix`::
+For simple projects that only need to build for a single architecture and a single set of dependencies.
+or create a `/ci.nix` or `/nix/ci.nix`::
+For projects that need to build for multiple architectures or versions, or where `default.nix` already
+has a special meaning that conflicts with the need for CI.
+
+3. Use pinned dependencies instead of `<nixpkgs>` or similar.
++
+Hercules CI uses an empty `NIX_PATH` to help with evaluation reprodicibility.
```

Comment:
I'd add here a copy-pasteable example for quickstart, i.e. build gnu hello world.

Reply:
I want to do that as part of a tutorial. It is a good alternative if you don't want to nixify. Maybe quickstart should be the goal of this guide and nixifying the subject of another guide. :+1: